src/Microsoft.OpenApi/Models/OpenApiReference.cs

```diff
@@ -299,5 +313,43 @@ internal void SetSummaryAndDescriptionFromMapNode(MapNode mapNode)
                 Summary = summary;
             }
         }
+
+        internal void SetJsonPointerPath(string pointer, string nodeLocation)
+        {
+            // Relative reference to internal JSON schema node/resource (e.g. "#/properties/b")
+            if (pointer.StartsWith("#/", StringComparison.OrdinalIgnoreCase) && !pointer.Contains("/components/schemas"))
+            {
+                ReferenceV3 = ResolveRelativePointer(nodeLocation, pointer);
+            }
+
+            // Absolute reference or anchor (e.g. "#/components/schemas/..." or full URL)
+            else if ((pointer.Contains('#') || pointer.StartsWith("http", StringComparison.OrdinalIgnoreCase)) 
```

Comment:
should this also support local storage paths? (either relative or absolute)
what about relative http paths?

Reply:
This hasn't been scoped out in the spec at the moment. I think once included, it can be implemented post GA

Reply:
CC @darrelmiller in case this was an oversight and needs to be completed by GA